apps/condo/domains/ai/schema/ExecutionAIFlowTask.test.js

```
@@ -32,11 +32,13 @@ const {
     FAULTY_FLOWISE_PREDICTION_RESULT,
     SUCCESS_FLOWISE_PREDICTION_RESULT,
 } = require('@condo/domains/ai/utils/testSchema/AIFlowTestingApps/FlowiseTestingApp')
+const { makeClientWithProperty } = require('@condo/domains/property/utils/testSchema')
 const {
     makeClientWithNewRegisteredAndLoggedInUser,
     makeClientWithSupportUser,
 } = require('@condo/domains/user/utils/testSchema')
```

ESLint

[error] There should be no empty line within import group

(import/order)